bootstrap/src/main/kotlin/jp/co/soramitsu/bootstrap/iroha/LazyIrohaAPIPool.kt

```diff
@@ -0,0 +1,38 @@
+package jp.co.soramitsu.bootstrap.iroha
+
+import jp.co.soramitsu.bootstrap.dto.IrohaConfig
+import jp.co.soramitsu.iroha.java.IrohaAPI
+import org.springframework.stereotype.Component
+import java.io.Closeable
+
+/**
+ * Lazy Iroha API pool
+ */
```

Comment:
This pool looks like a Dummy pool. It should have ability to check state of connection and remove connection if expired. This realisation may cause memory leaks. So should be upgraded in the future.

Reply:
Please take a look. It's fixed now.